src/Mobs/PathFinder.cpp

```diff
@@ -142,7 +142,7 @@ void cPathFinder::ResetPathFinding(cChunk &a_Chunk)
 	m_NoPathToTarget = false;
 	m_PathDestination = m_FinalDestination;
 	m_DeviationOrigin = m_PathDestination;
-	m_Path = cPath(a_Chunk, m_Source, m_PathDestination, 20, m_Width, m_Height);
```

Comment:
Sorry, my bad.
Ok, it calls the default assignment oprerator, which calls the assignment operator for all the fields of cPath, which, as far as I can tell, shouldn't leak. I think we should pinpoint the issue, which is possible in cPath, before merging this, to avoid further trouble.